### PR TITLE
Index page

### PR DESCRIPTION
**Pull request by Misori-simon**
# THE NEXT WEB homepage

## Intro

We have used several techniques like flexbox, position, float, and grid to layout our website and make it responsive. And it's about The New Web's Home Page.

## About the Project

This project is about building a responsive website using the various positioning and display values from the mobile view up to the desktop. The website has two breakpoints 676px and 1023px

### CODE REVIEW

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle:

## :wave: :wave: Hello!! Great job working on the project!!
:100: Great that you use GitHub pages
:100: Awesome that you have a licence
:100: Descriptive README
:100: Descriptive PR

## Required Changes
### [Mandatory] :red_circle: 

- [x] Footer should be outside the ```<main>```

- [x] Maintain alignment and indentation consistent through all the file. See inline comments

- [x] Latest Deals elements differ from original
image

- [x] Sections don't take the full width on larger screens 1440
image

- [x] Not all elements on navbar look like the original on table mode
image

- [x] latest funding only show 3 elements on tablet mode
image

- [x] Image distorting and overlaping text on tablet mode
image

- [x] Navbar doesn't look like original on mobile view
image

- [x] Latest funding show only 3 elements on mobile view
image

- [x] Image overlaping text on mobile view
image

- [x] Footer doesn't look as original on mobile view
image

### [OPTIONAL]  :large_blue_circle: 

- [ ] Maintain the CSS file DRY.

- [ ] For future project choose a meaningful name for the feature branch. Example:
**Actual**: ```index-page```
**Improved** :+1:  : ```thenextweb-clone```

### [Comments] :green_circle: 

- A great tool to test if the project looks as intended in different screen sizes is the browser media query inspector.

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,980 @@
...
+</span>
+<span>
+  Powered by <span>STACKPATH</span>
+</span>
+</div>
+</footer>
+  </main>
+  <!-- footer -->
+</body>
+</html>
```
Comment (jubaan) [inline index.html]: Fix indentation and alignment as required

**Review by jubaan — CHANGES_REQUESTED** [review]

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
### Request changes 😳
I am very happy to see this project and I can see you worked hard on this project but I am sorry to inform you that I cannot approve this project because it is not completed yet.😟 You need to work more on it. 
_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_
**Please have a look at the suggestions to get more understanding.**
- [x] Please use different images and text for each image. This will help you to show your project effectively. 

- [x] on the top nav-bar you didn't use actual search icon. 

- [x] Latest funding section should look like this.
chrome-capture (27)

- [x] Top nav-bar should be sticky in all screens. 

**Please make the changes and request re-review for the given project**
Stay safe and give me a smile 😇

### Happy Coding
---------------------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle:

Great job working on the previous required changes, there are still some issues that need to be addressed. 

## Required Changes
### [Mandatory] :red_circle: 

- [x] Latest funding doesn't match original in tablet view
image

- [x] Latest funding only shows 3 elements in mobile view
image

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Review by arslanbisharat — APPROVED** [review]
Congratslutons Approved 🎉
Hello Developer 👋
You are on 🔥
Thank you so much for making those changes and I am very happy to approve the project. You have done a great job. 👏
Please keep working hard and move forward with your curriculum.
Looking forward to seeing your next project. 😄
---------------------------------------------------------------------------------------------------------------
**To highlight**
- The cloned page is rendering as it should and meets all the project specifications.
-  CSS is clean.
-  The markup as well has no errors nor warnings to show. Good job with that.
-  The repo has followed all the guidelines there are to follow.
-  All the required tasks have been fulfilled in this project.
-  Congratulations on the splendid job👍
Now you can merge the branches and move forward with your curriculum. 🎆 

- [ ] Please use descriptive commit messages. 

[OPTIONAL] If you wish to delete your currently working branch please update the Live demo link too so that it works properly. 

_Keep working hard._ 

I am very excited to see your next project.

**Happy Coding**:smiley:
------------------------------------------------------------------------------
**Regards,
Arslan Bisharat**
[linkedin](https://www.linkedin.com/in/arslanbisharat/) slack: @arslanbisharat

**Pull request merged**